test/clj/game/cards/hardware_test.clj

```diff
@@ -860,7 +875,23 @@
         (let [inti (get-program state 0)]
           (is (not (nil? inti)) "Program was installed")
           (is (= (:title inti) "Inti") "Program is Inti")
-          (is (= (get-in @state [:runner :click]) 3) "Runner has 3 clicks left"))))))
+          (is (= (get-in @state [:runner :click]) 3) "Runner has 3 clicks left")))))
+  (testing "Clone Chip Heap Locked Test"
+    (do-game
+      (new-game {:corp {:deck [(qty "Blacklist" 1)  (qty "Wraparound" 5) ]}
+                 :runner {:deck ["Datasucker" (qty "Clone Chip" 2)]}})
+      (play-from-hand state :corp "Blacklist" "New remote")
+      (take-credits state :corp)
+      (trash-from-hand state :runner "Datasucker")
+      (rez state :corp (refresh (get-content state :remote1 0)))
+      (is (= 1 (count (:discard (get-runner)))) "Datasucker in heap")
+      (play-from-hand state :runner "Clone Chip")
+      (let [chip (get-hardware state 0)]
+        (card-ability state :runner chip 0)
+        (is (empty? (:prompt (get-runner))) "Clone Chip Prompt did not came up")
+        (is (= 1 (count (:discard (get-runner)))) "Datasucker in still in heap"))))
+  )
```

Comment:
:eyes:

Reply:
Are the eyes a request to fix parens new lines? or something else?

Reply:
Yep! just a silly way of noting that they're there.